docs/go-sql-instrumentation.md

```diff
@@ -0,0 +1,136 @@
+---
+id: go-sql-instrumentation
+title: Scope Go Agent SQL Instrumentation
+sidebar_label: SQL Instrumentation
+---
+
+
+## Instrumenting the SQL driver
+
+First, make sure the agent has been [installed](go-installation.md) in your Go application.
+
+The Scope Go agent provides a wrapper function to instrument an existing `database/sql` driver. 
+
+Currently, the instrumentation supports the following libraries:
+
+- Postgres (https://github.com/lib/pq)
+- MySql (https://github.com/go-sql-driver/mysql)
+
+To instrument the driver you need to register a new one using the wrapper:
+
+```go
+import (
+    "database/sql"
+    
+    "github.com/lib/pq"
+    "github.com/go-sql-driver/mysql"
+
+    scopesql "go.undefinedlabs.com/scopeagent/instrumentation/sql"
+)
+
+func main() {
+    // ...
+	
+	sql.Register("scope-postgres", scopesql.WrapDriver(&pq.Driver{}))
+    sql.Register("scope-mysql", scopesql.WrapDriver(&mysql.MySQLDriver{}))
+
+    // ...
+}
+```
+
+Also, by using `scopesql.WithStatements()` the sql statement values will be instrumented aswell. 
```

Comment:
```suggestion
Also, by using `scopesql.WithStatements()` the SQL statement values will be instrumented as well. 
```

And I would separate this into a different section, as it's not part of the basic installation. And also, statements are always captured right? The option is to enable statement _values_. The option has to be renamed.

Reply:
done, please check again.